documentation/integration_test_debugging_procedures.asciidoc

```diff
@@ -0,0 +1,120 @@
+
+= Integration test debugging procedures
+
+Debugging integration tests in the CobiGen project can be a complex process, as it involves multiple components working together. However, with the right tools and techniques, you can effectively debug your tests and resolve any issues that may arise.
+
+=====  •  System.out.println()
```

Comment:
You could use `==` instead of `===== •` throughout the file. Especially with more text it is hard to see the titles.